### PR TITLE
fix Drag and Drop of Network modifications list + feat Move progession status in subtitle

### PR DESCRIPTION
**Pull request by sBouzols**
Signed-off-by: sBouzols <sylvain.bouzols@gmail.com>

### CODE REVIEW

**Review by bendaamerahm — APPROVED** [review]

File: src/components/graph/menus/network-modification-node-editor.js [hunk 1]
```diff
@@ -55,9 +55,16 @@ import {
...
+    listContainer: {
+        overflowY: 'auto',
+        display: 'flex',
+        flexDirection: 'column',
+        flexGrow: 1,
+    },
     list: {
         paddingTop: theme.spacing(0),
-        overflowY: 'auto',
+        // backgroundColor: 'red',
```
Comment (etiennehomer) [inline src/components/graph/menus/network-modification-node-editor.js:66]: remove ?

File: src/components/graph/menus/network-modification-node-editor.js [hunk 2]
```diff
@@ -334,6 +341,7 @@ const NetworkModificationNodeEditor = ({ selectedNode }) => {
             // (work for all users)
             // specific message id for each action type
             setMessageId(messageId);
+            console.debug('SBO notif: ', messageId);
```
Comment (etiennehomer) [inline src/components/graph/menus/network-modification-node-editor.js:344]: remove ?

**Review by etiennehomer — DISMISSED** [review]

File: src/components/graph/menus/network-modification-node-editor.js (hunk 2 above)
Comment (etiennehomer) [inline src/components/graph/menus/network-modification-node-editor.js:344]: remove

**Review by bendaamerahm — APPROVED** [review]

File: src/components/graph/menus/network-modification-node-editor.js [hunk 3]
```diff
@@ -362,6 +370,11 @@ const NetworkModificationNodeEditor = ({ selectedNode }) => {
 
     useEffect(() => {
         if (studyUpdatedForce.eventData.headers) {
+            console.debug(
+                'SBO studyUpdatedForce: ',
+                studyUpdatedForce.eventData.headers
```
[the remark this answers is not in the document]
Comment (sBouzols) [reply src/components/graph/menus/network-modification-node-editor.js]: removed

File: src/components/graph/menus/network-modification-node-editor.js (hunk 2 above)
Comment (sBouzols) [reply src/components/graph/menus/network-modification-node-editor.js]: removed

**Review by bendaamerahm — APPROVED** [review]

**Review by JacquesBorsenberger — APPROVED** [review]
I'm still  wondering if it is better to have 
`{renderSomthing()}`
or
`<Something/>`

**Review by laurgarn — APPROVED** [review]

**Pull request merged**